docs/maintainers/maintainers-guide.md

```diff
@@ -43,25 +43,25 @@ There are two ways to add repositories to your Repository Insight Page:
 
 ### Using Your Repository Insights Page
 
-Once you have connected your repositories, you will be redirected to your Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
+Once you have connected your repositories, you will be redirected to your Repository Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
 
 There are three tabs that provide you with more information on these repositories:
 
 #### Reports
```

Comment:
I don't see "Reports" tab in on the Repository Insight Page. There are only "Dashboard", "Contributors" and "Activity".

![Screenshot 2024-02-28 215744](https://github.com/open-sauced/docs/assets/45172775/634ff737-6a81-4cdd-9f7e-84d33e06489c)

Reply:
Hi @adiati98. Repositories section does sound clearer.

Reply:
Repositories Dashboard sounds great!

Also, I'll remove Reports altogether with this PR. 👍